src/Storage/Commands.Storage/Blob/Cmdlet/StartAzureStorageBlobIncrementalCopy.cs

```diff
@@ -0,0 +1,455 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+
+namespace Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet
+{
+    using Commands.Common.Storage.ResourceModel;
+    using Microsoft.WindowsAzure.Commands.Common.Storage;
+    using Microsoft.WindowsAzure.Commands.Storage.Common;
+    using Microsoft.WindowsAzure.Commands.Storage.Model.Contract;
+    using Microsoft.WindowsAzure.Storage;
+    using Microsoft.WindowsAzure.Storage.Blob;
+    using System;
+    using System.Management.Automation;
+    using System.Security.Permissions;
+    using System.Threading.Tasks;
+
+    [Cmdlet(VerbsLifecycle.Start, StorageNouns.IncrementalCopyBlob, SupportsShouldProcess = true, DefaultParameterSetName = ContainerParameterSet),
+       OutputType(typeof(AzureStorageBlob))]
```

Comment:
@blueww I may have missed it in this cmdlet, but it doesn't look like the cmdlet writes any `AzureStorageBlob` object to the pipeline, even though the output type specifies that it does. It this intended?